### PR TITLE
Portal = Contact

### PR DESCRIPTION
**Pull request by GalinaRuban**
https://trello.com/c/g7EFu4PW/517-portal-contact-automation, regular test (not POM)

### CODE REVIEW

File: src/test/java/ContactPageTest.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        fillContactForm(FIELD_INPUTS[1], FIELD_INPUTS[2], FIELD_INPUTS[3], FIELD_INPUTS[4]);
+        ProjectUtils.click(getDriver(), getDriver().findElement(SAVE_BTN));
+
+        List<WebElement> trs = getDriver().findElements(By.xpath("//table[@id='pa-all-entities-table']/tbody/tr"));
+        Assert.assertEquals(trs.size(), 1);
+
+        List<WebElement> allLines = getDriver().findElements(By.xpath("//td"));
+        for (int i = 1; i < allLines.size() - 1; i++) {
+            if (i == 5) {
+                Assert.assertEquals(formatDate(FIELD_INPUTS[i]), allLines.get(i).getText());
```
Comment (Stan575) [inline src/test/java/ContactPageTest.java:76]: Actual value should be passed as first argument, expected value should be second argument

**Review by Stan575 — APPROVED** [review]

**Review by oooleks — COMMENTED** [review]
There is GMT time zone that used by Portal application, but not "Europe/Warsaw", so to properly generate current date in format dd/mm/yyyy you may use:
SimpleDateFormat dateFormatGmt = new SimpleDateFormat("dd/MM/yyyy");
dateFormatGmt.setTimeZone(TimeZone.getTimeZone("GMT"));
String time = dateFormatGmt.format(new Date());
instead of 
private static final String CURRENT_DATE = String.valueOf(LocalDate.now(ZoneId.of("Europe/Warsaw")));
Otherwise assert may fail.

File: src/test/java/ContactPageTest.java (hunk 1 above)
Comment (GalinaRuban) [reply src/test/java/ContactPageTest.java]: Done with the new commit.

**Review by oooleks — COMMENTED** [review]
- No need to call getDriver() method all the time in your test while you can use WebDriver driver = getDriver(); and use the variable "driver" everywhere in your test (performance issue);
- for MESSAGE you can use ProjectUtils.createUUID() method to generate uniquе String value;
- there is no much point to create methods with one line of code to use it only once (navigateToContact() and createNewRecord());
- fillContactForm could be implemented as a loop;
- not necessary empty 64 line;
- instead of List<WebElement> allLines = getDriver().findElements(By.xpath("//td")); (line 73) which finds all 8 elements of record you can use xpath: "//tbody//a/div" to find 6 related record fields elements to assert in loop which could start from 0 but not from 1;

**Review by oooleks — APPROVED** [review]

**Pull request merged**